superset/sql_parse.py

```diff
@@ -143,15 +146,20 @@ def as_create_table(self, table_name, overwrite=False):
         exec_sql += f"CREATE TABLE {table_name} AS \n{sql}"
         return exec_sql
 
-    def __extract_from_token(self, token, depth=0):
+    def __extract_from_token(self, token: Token):
```

Comment:
I removed `depth`, as it had no functional utility (I'm assuming it was used during debugging during initial development). Could potentially be added back with a `max_depth` attribute to make sure it doesn't go too deep.